specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -2674,6 +2677,9 @@
           }
         }
       },
+      "required": [
```

Comment:
Are required fields actually changing in the API or is this just to fulfill swagger validation?

Reply:
just to fulfill the swagger validation, nothing changed in the API. "kind" now must be a required field, this is probably a new validation that wasn't before